tests/testproject/TestProject.IntegrationServiceA/SqlServer/SqlServerExtensions.cs

```diff
@@ -14,13 +15,21 @@ private static async Task<IResult> VerifySqlServerAsync(SqlConnection connection
     {
         try
         {
-            await connection.OpenAsync();
+            var policy = Policy
```

Comment:
Why does this need retries?

Reply:
Because the SqlServer container takes like ~30-40 seconds to start up. If you run this test by itself (or it is the first one being executed), the server isn't available and the test fails.